.github/workflows/cml.yaml

```diff
@@ -1,23 +1,29 @@
 name: CML Report
-on: push
+on: [push, pull_request]
+concurrency:
```

Comment:
Q: what does it give us? Can we keep it simple(r) for this repo (even if we hit some edge cases)?

Reply:
- right now, we can't keep it simpler for the desired behaviour
- if we're willing to drop/change the desired behaviour, we can simplify this
- this PR currently demonstrates how hard it is to use GHA/CML to do what should be extremely simple. I intend it as something we should write a technical article about. I do not think we should merge this here nor into https://github.com/iterative/example-repos-dev. It is wayyy to complex.

Reply:
WDYT about the ["**Previously in `c3b04d0`**" version](https://github.com/iterative/example-get-started/pull/31#issue-1411746987)?

Reply:
I think let's step back and do the simplest one - PR, no main/master, no ifs, no cancellation, etc. We are going a bit too far with this tbh.